lib/deaf_grandma.rb

```diff
@@ -14,14 +15,21 @@ def run!
     loop do
```

Comment:
Instead of a loop do with a break inside of the body, you could use a structured loop that specifies its end condition. Consider using a while loop, or for extra fun, use until. =)

``` 
while conditional do
  stuff
end

until conditional do
  stuff
end
```